solutions/dashboard/tools/jenkins/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3.10.5
+
+COPY . /app
+
+WORKDIR /app
+
+RUN pip3 install pymongo==4.2.0
```

Comment:
Can we create requirements.txt and install all packages in single command This will reduce docker image layers.

Reply:
Yes, we can do that

Reply:
Please plan this as part of the deployment task.